## Alignment 
`GSE123507_alignment.run`
This takes the path `$DIR` and processes all the files under it and outputs to output directory `$ODIR`, named with suffix
It uses the same index for C. elegans stuff, so can just link to the old one if you already have one.

```bash
#!/bin/bash
#SBATCH --time=24:00:00
#SBATCH --account=def-taubert
#SBATCH --job-name=GSE123507_alignment
#SBATCH --output=%x-%j.out
#SBATCH --mem=256G
#SBATCH --cpus-per-task=32
#SBATCH --output=%x-%j.out

module load StdEnv/2020 gcc/9.3.0 openmpi/4.0.3
module load salmon/1.4.0

DIR="/home/jyan/scratch/GSE123507/trimmed_reads"
ODIR="/home/jyan/scratch/GSE123507/alignment"
IDIR="/home/jyan/projects/def-taubert/jyan/RNAseq/index/salmon_index"
SUFFIX="quant"

# single end only
# loop over filename.fasta.gz in $DIR, output to $ODIR under filename_quant

for filename in $DIR/*; do 
 basename=${filename##*/}; #strip path
 basename=${basename%%.*}_${SUFFIX}; #strip file extension/s and add suffix
 #echo $basename;
 salmon quant -i $IDIR -l A -r ${filename} -o $ODIR/${basename} --gcBias --validateMappings --threads 30
done
```


To get alignment files and folders:

```bash
rsync -a jyan@cedar.computecanada.ca:~/scratch/GSE123507/alignment/ ~/Desktop/alignment

```
